recipes/dexplot/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "dexplot" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0082db18cde92e877935aaa53e2c48692b00b69e1c6d6aad7bd2576e44b29e68
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
```

Comment:
Because you are importing from `dexplot` in the `setup.py`, you're likely going to need the `run` dependencies under `host` as well. 

Also please change `python` to `python >=3.6` under `host` and `run`.

Reply:
Thanks for mentioning this. Didn't know the issues caused by importing `__version__` from `__init__.py` caused. I've changed all of my setup.py files to read the version from the file instead.